### PR TITLE
Remove `mux` alias in Bash completion file

### LINKED ISSUE #868: Use of `alias` in the completion file
**Describe the bug**

I noticed that in `tmuxinator.bash`, there is a line:

```sh
alias mux="tmuxinator"
```

and also in zsh:

```sh
abbr --add mux "tmuxinator"
```

**Expected behavior**
I expect no completion file to automatically create any aliases - It's convention not to, and is kind of sneaky, because it's not really expected. It is also inconstant because there is no such alias in the Fish completions file. Please remove this? It is a breaking change, but of the people that actually use this alias can add it to their own bashrc or zshrc

**Environment**
- Any Bash environment

### PR DESCRIPTION
**Pull request by hyperupcall**
It should be removed because:

- Aliases in completion files are unconventional
- This will conflict with user-defined aliases in unexpected and hard-to-debug ways
- It is not consistent with completions for the other shells that we support (zsh)
- Is confusing

Closes #868

### CODE REVIEW

**Review by akofink — APPROVED** [review]
Makes sense to me and matches the other shell completion files, as you point out. I personally do have this alias explicitly defined in my `.zshrc`.

**Comment by akofink** [comment]
Would you mind adding an entry to the `CHANGELOG.md`?

We could also probably remove this from `tmuxinator.fish`.

**Comment by hyperupcall** [comment]
@akofink Good idea, I've added an entry to the changelog and removed it from Fish as well

**Review by akofink — APPROVED** [review]

**Comment by akofink** [comment]
Any other feedback from @tmuxinator/tmuxinator on this one?
The change looks ready to merge to me, but I can't say I'm an expert on these shell completions.

**Comment by akofink** [comment]
Hey there, apologies for not merging this earlier!

Would you mind addressing the conflicts in the CHANGELOG.md? Then we should be good to merge. 🙏

**Comment by hyperupcall** [comment]
@akofink No worries, addressed the merge conflicts

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 ### Misc
 - Fix typos and remove extra whitespace
 - Set pane title if provided in config file
+- Remove `mux` alias from Fish and Bash completions
```
Comment (akofink) [inline CHANGELOG.md:8]: This change should be under the `## Unreleased` header, above 3.1.2.

**Comment by akofink** [comment]
This was merged as https://github.com/tmuxinator/tmuxinator/pull/906

Thanks again, @hyperupcall!

**Pull request closed without merging**